examples/skr/aks/README.md

```diff
@@ -1,4 +1,8 @@
-# Microsoft Secure Key Release (SKR) AKS Example
+# NOTE This example is for te Kata based AKS preview
+#      Today the supported GA solution for Kubernetes is to use virtual nodes.
```

Comment:
We should link the readme to VN2

Reply:
I had failed to add a top level readme which has some links. I added it here too.